pyQuARC/code/downloader.py

```diff
@@ -98,7 +99,11 @@ def download(self):
 
         # constructs url based on concept id
         url = self._construct_url()
-        response = requests.get(url)
+        token = os.environ.get("AUTH_TOKEN")
+        headers = None
+        if token:
+            headers = {"Authorization": f"Bearer {token}"}
+        response = requests.get(url, headers=headers)
```

Comment:
there are many other requests to the cmr, can you create a utility function that does this and then call that function everywhere a `requests.get()` is called?

Reply:
Resolved [here](https://github.com/NASA-IMPACT/pyQuARC/pull/206/commits/963d668b869a204efb1c618ebc386363546f4238)

Reply:
can you do it for all the `requests.get()`s, not just those two

Reply:
Addressed [here]([598228f](https://github.com/NASA-IMPACT/pyQuARC/pull/206/commits/598228fdc2bca04c4cc65b1c7207e2647bf43d93))